bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -521,7 +521,11 @@ spec:
           export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/var/run/ovn/ovn-nbctl.pid \
             --detach \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
-            --db "{{.OVN_NB_DB_LIST}}")
+            --db "{{.OVN_NB_DB_LIST}} --log-file=/run/ovn/ovn-nbctl.log")
+
+          # include nbctl daemon logging, allow for ovn-nbctl to create the log file
+          sleep 1
+          tail -f /run/ovn/ovn-nbctl.log &
```

Comment:
add `--retry` to the `tail` args and then it will work even if "sleep 1" isn't long enough.

(You could theoretically get rid of the "sleep 1" entirely, but then tail would log a pointless warning, so it's best to have both.)